hydra_agent/graphutils.py

```diff
@@ -0,0 +1,95 @@
+from redisgraph import Node, Edge, Graph
+
+
+class GraphUtils:
+
+    def __init__(self, redis_proxy, graph_name='apidoc'):
+        self.redis_proxy = redis_proxy
+        self.redis_connection = redis_proxy.get_connection()
+        self.graph_name = graph_name
+        self.redis_graph = Graph("apidoc", redis_proxy)
+
+    def read(self, match, ret, where=None):
+        """
+        Run query to read nodes in Redis and return the result
+        :param match: Relationship between queried entities.
```

Comment:
params shall have types, i.e.:
* `:param match (<type>): <description>`

Reply:
I took a look at how it's done in hydrus and it's currently something like this:

![image](https://user-images.githubusercontent.com/11739405/59671235-9f5c8d00-91bd-11e9-9dc8-0814f3052e5d.png)

Should I proceed with them like this or put them in the docstring?

Reply:
Right, better use type annotations